packages/cardano-services/src/ChainHistory/DbSyncChainHistory/ChainHistoryBuilder.ts

```diff
@@ -1,4 +1,4 @@
-// cSpell:ignore descr
+// cSpell:ignore descr timelock
```

Comment:
Please remove all cSpell comments or set it up as a check in the repo

Reply:
cSpell is a VSCode extension. What is the problem?
Those who have it installed, can get benefit from it; those who don't are not impacted by the comments.

Reply:
Problem is that it's specific to your dev environment. These comments don't make sense for the repo in general. My suggestion here is to either:
- add a "cspell" npm script and run it in CI
- or don't commit those comments

Reply:
Yes, I understand it's relative to my dev environment, but I can't see which problem those comments are originating.
Yes, if could be a good idea to add a repository level additional check, but it is OT from this PR.

Reply:
Ok let's make a deal: keep it but follow-up with a new PR to add this check :)

Reply:
The comments can be removed altogether.
Workspace ignored words are stored in `.vscode/settings.json`, and the `.vscode` folder is gitignored.
<img width="777" alt="image" src="https://github.com/user-attachments/assets/d66b713d-dbc0-43dc-a075-012863f7306f">